testsuite/gnat2goto/tests/deferred_constant_use/test.opt

```diff
@@ -0,0 +1 @@
+ALL XFAIL cbmc returns failed verification (calling function without body)
```

Comment:
that seems quite worrying, considering there is no bodyless function in here

Reply:
There is: `Inc`. The error comes from running CBMC on `use_deferred_const.adb` which calls `Inc` but `deferred_const.ads` does not contain the body.

Reply:
I thought I had left a comment about this in an earlier PR - but I can't find it.  May be I forgot to press the comment button.  When we analyse single add files by cbmc rather than a complete program, subprograms declared in package bodies are not analysed.  This is an issue we have to discuss with Martin as to the best way to handle the problem.